### ISSUE TITLE
Add languages field

### ISSUE THREAD

**Issue by isacmoura** [salomonelli/best-resume-ever#329]
### Case
- FEATURE REQUEST

## Issue
Include field for languages that we domain

## Info
- Operating System: Linux
- Node-Version: 10.15.0

**Comment by pubkey** [comment]
Yes this is a good idea. PR will be welcomed.

**Comment by isacmoura** [comment]
I tried to include but one doubt I had been: the project don't have a major model? Have I to include this functionality in all templates?

**Comment by salomonelli** [comment]
Yes, we should adapt the templates, otherwise it may be confusing and inconsistent. PRs welcome :)
